src/common/adapters/jira.test.js

```diff
@@ -1,5 +1,3 @@
-import { URLSearchParams } from 'whatwg-url';
-
```

Comment:
I think we also don't need to have the package as a dependency (since it's polyfilled by JSDOM).